src/js/workshop-comments.js

```diff
@@ -124,15 +124,19 @@ function disableFields(disabled) {
 	// When a UIO theme is selected, the background color will remain the same as what is set by the theme.
 	const backgroundColor = disabled ? "#f3f3f3" : "#fff";
 
-	// Disable/enable and set background color of post a comment button
+	// Disable/enable and set background color of post a comment button.
```

Comment:
Add quotes around "post a comment" so this sentence doesn't confuse us after a few months.